src/_data/sidenav.yml

```diff
@@ -65,9 +65,9 @@
             - title: Introduction
               permalink: /docs/development/ui/animations
               match-page-url-exactly: true
-            - title: Animations overview
+            - title: Overview
               permalink: /docs/development/ui/animations/overview
-            - title: Animations tutorial
+            - title: Tutorial
```

Comment:
@sfshaza2 - note the two changes in the sidenav entry names

Reply:
That's fine.